sources/packages/api/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt-files/disbursement-receipt-file-detail.ts

```diff
@@ -0,0 +1,123 @@
+import { DisbursementReceiptRecord } from "./disbursement-receipt-file-record";
+
+/**
+ * Model to parse the grant field of disbursement file detail record.
+ */
+export class DisbursementReceiptGrant {
+  constructor(
+    public readonly grantType: string,
+    public readonly grantAmount: string,
+  ) {}
+}
+
+/**
+ * Disbursement receipt detail record which has the receipt details for disbursements sent.
+ * The document number which is present in each of the receipt detail record connects the
+ * disbursement sent with the receipt received.
+ */
+export class DisbursementReceiptDetail extends DisbursementReceiptRecord {
+  constructor(line: string, lineNumber: number) {
+    super(line, lineNumber);
+  }
+
+  //Number of occurrences of the grant field.
```

Comment:
The comment is good but I would recommend adding it in a way that VS Code will be able to recognize it.